lib/external-housing-available/external-housing-available.models.js

```diff
@@ -0,0 +1,30 @@
+const Joi = require('joi');
+
+module.exports = {
+  db: (sequelize, Sequelize) => {
+    const ExternalHousingAvailable = sequelize.define('ExternalHousingAvailable', {
+      id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+        primaryKey: true
+      },
+      author: Sequelize.STRING,
+      title: Sequelize.STRING,
+      address: Sequelize.STRING,
+      location: Sequelize.STRING,
+      beds: Sequelize.INTEGER,
+      baths: Sequelize.DECIMAL,
+      contact: Sequelize.STRING,
+      body: Sequelize.TEXT,
+      status: Sequelize.STRING,
+      url: Sequelize.TEXT,
+      details: {
+        type: Sequelize.JSONB,
+        defaultValue: {},
+      },
+    });
+    return ExternalHousingAvailable;
```

Comment:
we'll still need to define associations for this model. We can go over this as a group. I can't remember if I told you all to hold off on that or give it a shot. my apologies.